### ISSUE TITLE
Message about temporary admin user is confusing.

### ISSUE THREAD

**Issue by ssilvert** [keycloak/keycloak#33965]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

We now have a warning message about the initial "temporary" admin user when Keycloak is first installed.  However, users are confused as to how they can create a permanent admin user.

See https://mail.google.com/mail/u/0/?tab=cm#search/label%3Ainbox+label%3AKeycloak-Discussion+-label%3AMentioned/FMfcgzQXJZzBZlSggRvpgLzQrrNZCBzm

### Version

26.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

At the very least, the error message should contain a link with instructions on how to create the initial admin user.

Even better would be to create a wizard for this.  And at the end, it would allow deletion of the temporary user.

### Actual behavior

See above.

### How to Reproduce?

See above.

**Comment by edewit** [comment]
there is a wizard for this already it's called the welcome screen

**Comment by jonkoops** [comment]
The welcome screen creates the temporary admin user, not a permanent admin user.

**Comment by edewit** [comment]
Oh, I thought this was only shown when you use the Environment variables to create a user, in that case, I don't even know how to create a permanent admin user.
@ssilvert could you maybe state what you want the message to say, because you put in a link to your inbox

**Comment by ssilvert** [comment]
> Looks like this is causing some confusion amongst users already, leading them to create new admin users that are not fully functional, see #35115.

Yes, and we really aren't sure how to address this.  We have the idea making a wizard that helps set up a proper admin user suitable for a production environment.  But it's unclear exactly what that new admin user would look like.  There are any number of ways that the user could have security that is better than username/password.  But the best choice is different for each environment.

I'm starting to think that the best idea is to link to some documentation instead of creating a wizard.

**Comment by jonkoops** [comment]
> I'm starting to think that the best idea is to link to some documentation instead of creating a wizard.

This might be a good starting point yeah, it can always be enhanced further down the line.

**Comment by jonkoops** [comment]
@ssilvert FYI, we closed the aforementioned issue since @mposolda produced some steps to resolve it (https://github.com/keycloak/keycloak/issues/35115#issuecomment-2497342910).

**Comment by edewit** [comment]
I don't think we need a wizard for this issue just some documentation. What we need for a good admin user is a password and some otp and a verified email. All of this can be done with the normal user flow.

**Comment by jonkoops** [comment]
Yeah, as it is right now an initial admin user can simply be replaced with one that has all the same permissions, and you don't even need to set up 2FA. This seems like it is kinda wrong tbh, ideally a warning would simply be shown to any user that is has the `admin` role, but has not set up 2FA, so even the initial admin user would qualify as long as it is set up properly later on (no need to remove the user).

IMHO as the feature is now it seems counterproductive, it does not reach the goal of making a user secure their account better.
